doc/neps/index.rst

```diff
@@ -6,8 +6,7 @@ This page provides an overview of development priorities for NumPy.
 Specifically, it contains a roadmap with a higher-level overview, as
 well as NumPy Enhancement Proposals (NEPs)—suggested changes
 to the library—in various stages of discussion or completion.
-See `NEP 0 <https://github.com/numpy/numpy/blob/main/doc/neps/nep-0000.rst>`__
-for more informations about NEPs.
+See :doc:`nep-0000` for more informations about NEPs.
```

Comment:
This of course will also change the link wording to include the title of the NEP. If there's a strong preference to preserve the exact link rendering, please don't hesitate to do so (e.g. ``:doc:`NEP 0 <nep-0000>``)